tasklist/client/src/App.tsx

```diff
@@ -38,22 +38,25 @@ const Wrapper: React.FC = () => {
 
 const router = createBrowserRouter(
   createRoutesFromElements(
-    <Route path="/" element={<Wrapper />}>
+    <Route path="/" element={<Wrapper />} ErrorBoundary={ErrorWithinLayout}>
```

Comment:
what are these changes for?

Reply:
The error boundaries weren't on the correct levels to show the proper error page, so I fixed it since I was migrating those components already.